<?php

namespace ContainerB8dpxV3;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_Ti3yw8fService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.Ti3yw8f' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.Ti3yw8f'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'passwordHasher' => ['privates', 'security.user_password_hasher', 'getSecurity_UserPasswordHasherService', true],
            'user' => ['privates', '.errored..service_locator.Ti3yw8f.App\\Entity\\UserApp', NULL, 'Cannot autowire service ".service_locator.Ti3yw8f": it references class "App\\Entity\\UserApp" but no such service exists.'],
        ], [
            'passwordHasher' => '?',
            'user' => 'App\\Entity\\UserApp',
        ]);
    }
}
